example_pms_sensor.py

```diff
@@ -0,0 +1,24 @@
+from sensors.Pms import Pms
+
+def main():
+	DEVICE = '/dev/ttyAMA0'
+	TIMEOUT = 3
+	BAUD_RATE = 9600
+	PIN_ENABLE = 22
```

Comment:
PIN_ENABLE and PIN_RESET are unused(?)

Remove them.

Reply:
removed